docs/core/setup/index.md

```diff
@@ -0,0 +1,453 @@
+
+::: zone pivot="os-windows"
+
+```console
+dotnet --list-sdks
+
+2.1.500 [C:\program files\dotnet\sdk]
+2.1.502 [C:\program files\dotnet\sdk]
+2.1.504 [C:\program files\dotnet\sdk]
+2.1.600 [C:\program files\dotnet\sdk]
+2.1.602 [C:\program files\dotnet\sdk]
+2.2.101 [C:\program files\dotnet\sdk]
+3.0.100 [C:\program files\dotnet\sdk]
+```
+
+::: zone-end
+
+::: zone pivot="os-linux"
+
+```bash
+dotnet --list-sdks
+
+2.1.500 [/home/user/dotnet/sdk]
+2.1.502 [/home/user/dotnet/sdk]
+2.1.504 [/home/user/dotnet/sdk]
+2.1.600 [/home/user/dotnet/sdk]
+2.1.602 [/home/user/dotnet/sdk]
+2.2.101 [/home/user/dotnet/sdk]
+3.0.100 [/home/user/dotnet/sdk]
+```
+
+::: zone-end
+
+::: zone pivot="os-macos"
+
+```bash
+dotnet --list-sdks
+
+2.1.500 [/usr/local/share/dotnet/sdk]
+2.1.502 [/usr/local/share/dotnet/sdk]
+2.1.504 [/usr/local/share/dotnet/sdk]
+2.1.600 [/usr/local/share/dotnet/sdk]
+2.1.602 [/usr/local/share/dotnet/sdk]
+2.2.101 [/usr/local/share/dotnet/sdk]
+3.0.100 [/usr/local/share/dotnet/sdk]
+```
+
+::: zone-end
+
+You can also see what runtimes are installed with the command `dotnet --list-runtimes`.
+
+## Dependencies
+
+Depending on the OS you're using, you may need to install additional dependencies.
+
+::: zone pivot="os-windows"
+
+<!-- markdownlint-disable MD025 -->
+<!-- markdownlint-disable MD024 -->
+
+# [.NET Core 3.0](#tab/netcore30)
+
+The following Windows operating systems are supported with .NET Core 3.0:
+
+> [!NOTE]
+> A `+` symbol represents the minimum version.
+
+| OS                            | Version                        | Architectures   |
+| ----------------------------- | ------------------------------ | --------------- |
+| Windows Client                | 7 SP1+, 8.1                    | x64, x86        |
+| Windows 10 Client             | Version 1607+                  | x64, x86        |
+| Windows Server                | 2012 R2+                       | x64, x86        |
+| Nano Server                   | Version 1803+                  | x64, ARM32      |
+
+For more information about .NET Core 3.0 supported OS, distributions, and lifecycle policy, see [.NET Core 3.0 Supported OS Versions](https://github.com/dotnet/core/blob/master/release-notes/3.0/3.0-supported-os.md).
+
+# [.NET Core 2.2](#tab/netcore22)
+
+The following Windows operating systems are supported with .NET Core 2.2:
+
+> [!NOTE]
+> A `+` symbol represents the minimum version.
+
+| OS                            | Version                        | Architectures   |
+| ----------------------------- | ------------------------------ | --------------- |
+| Windows Client                | 7 SP1+, 8.1                    | x64, x86        |
+| Windows 10 Client             | Version 1607+                  | x64, x86        |
+| Windows Server                | 2008 R2 SP1+                   | x64, x86        |
+| Nano Server                   | Version 1709+                  | x64, ARM32      |
```

Comment:
Nano Server 1709 is EOL so it is technically not supported.  I think `1803+` would be more appropriate here.